packages/utils/examples/utils/utils.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import {
+	FilterModule,
+	LabelsModule,
+	WindowModule,
```

Comment:
Suggestion: if you already import these modules, maybe you should add the documentation too?

Reply:
I removed them again, so they can be separate PR's.